static/js/containers/TermsOfServicePage.js

```diff
@@ -31,25 +36,147 @@ class TermsOfServicePage extends React.Component {
   }
 
   render() {
-    return <div>
-      <h3>
-        MIT Micromasters Terms of Service
-      </h3>
-
-      <p>
-        Please read through the terms of service. To enroll in MIT Micromasters you
-        must agree to the terms of service.
-      </p>
+    const profile = this.props.userProfile.profile;
+    let linkToEdxTermsOfService = (
+      <a href="https://www.edx.org/edx-terms-service" target='_blank'>https://www.edx.org/edx-terms-service</a>
+    );
+    let imageUrl = `${SETTINGS.edx_base_url}/static/images/profiles/default_120.png`.
+    //replacing multiple "/" with a single forward slash, excluding the ones following the colon
+    replace(/([^:]\/)\/+/g, "$1");
```

Comment:
It removes duplicate slashes from the user image URL. But, this code already exists elsewhere in the project and we should create a component to share it if we are using it twice